docs/antrea-node-network-policy.md

```diff
@@ -0,0 +1,116 @@
+# Antrea Node NetworkPolicy
+
+## Table of Contents
+
+<!-- toc -->
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Usage](#usage)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+## Introduction
+
+Node NetworkPolicy is designed to secure the Kubernetes Nodes traffic. It is supported by Antrea starting with Antrea v1.15.
+
+This guide demonstrates how to configure Node NetworkPolicies.
```

Comment:
I feel no need to start a new paragraph for this sentence.

Reply:
Done